packages/components/src/components/select/select.stories.ts

```diff
@@ -71,600 +57,266 @@ export default {
-              { type: 'attribute', name: 'label', value: 'Required' },
-              { type: 'attribute', name: 'name', value: 'required field' }
-            ],
-            args
-          })}
-        </div>
-        <div class="mb-6">
-          ${generateTemplate({
-            constants: [
-              ...sharedConstants,
-              { type: 'attribute', name: 'label', value: 'Required Multiple' },
-              { type: 'attribute', name: 'name', value: 'required multiple field' },
-              multipleConstant
-            ],
-            args
-          })}
-        </div>
-        <div class="mb-8">
-          ${generateTemplate({
-            constants: [
-              ...sharedConstants,
-              { type: 'attribute', name: 'label', value: 'Required Multiple w/ Tags' },
-              { type: 'attribute', name: 'name', value: 'required multiple tags field' },
-              multipleConstant,
-              { type: 'attribute', name: 'useTags', value: true }
-            ],
-            args
-          })}
-        </div>
-        <sd-button type="submit">Submit</sd-button>
-      </form>
-      <script>
-        function handleSubmit(event) {
-          const form = document.querySelector('#testForm');
-          const sdSelects = Array.from(document.querySelectorAll('sd-select'));
-
-          const isValid = sdSelect => sdSelect.checkValidity();
-
-          if (sdSelects.every(isValid)) {
-            event.preventDefault(); // Prevent the default form submission behavior
-
-            const formData = new FormData(form);
-            const formValues = Object.fromEntries(formData);
-
-            alert('Form submitted successfully with the following values: ' + JSON.stringify(formValues, null, 2));
-          }
-        }
-
-        document.querySelector('#testForm').addEventListener('submit', handleSubmit);
-      </script>
-    `;
-  }
+export const MaxOptionsVisible = {
+  render: () => html`
+    <div class="w-[300px] h-[500px]">
+      <sd-select
+        size="lg"
+        label="Label"
+        placement="bottom"
+        multiple=""
+        value="option-1 option-2 option-3"
+        useTags
+        max-options-visible="2"
+      >
+        <sd-option value="option-1">Option 1</sd-option>
+        <sd-option value="option-2">Option 2</sd-option>
+        <sd-option value="option-3">Option 3</sd-option>
+        <sd-option value="option-4">Option 4</sd-option>
+        <sd-option value="option-5">Option 5</sd-option>
+      </sd-select>
+    </div>
+  `
 };
 
 /**
- * 1. You can use the `setCustomValidity` method to set a custom validation message. This will override any native validation messages.
- * 2. Set an empty string to clear the custom validity and make the input valid.
- * 3. To show the validation message, call the `reportValidity` method. Originally this would show a native validation bubble, but we show the error messages inline.
+ * Use the `required` attribute to make the select required.
```

Comment:
Use the ”required” attribute to mark the element as required. This can be used for form validation purposes.

Reply:
Updated